main.gms

```diff
@@ -222,7 +222,7 @@ parameters
   cm_iteration_max          "number of Negishi iterations"
   c_solver_try_max          "maximum number of inner iterations within one Negishi iteration (<10)"
   c_keep_iteration_gdxes    "save intermediate iteration gdxes"
-  cm_keep_presolve_gdxes    "save gdxes for all regions/solver tries/nash iterations for debugging"
+  cm_keep_presolve_gdxes    "save gdxes for all regions, solver tries and nash iterations for debugging"
```

Comment:
Michaja, I really don't understand your reaction. I passed hours last year until I found out that a careless `/` in the description of a new switch [here](https://github.com/remindmodel/remind/commit/e4b5beb2860b020ac5e53939ccbc8b9570dcd2ba#diff-475ac83ca355b537a8b9c25d17290cea6bb65d000e13fdf7df4632520155f4e2R710) caused several lines of code to vanish. And this *had* a substantial impact on REMIND performance, namely breaking it. You are lucky because you introduced two `/` at the same time :)

I just had another look at main.gms and saw that also [in this line](https://github.com/remindmodel/remind/blob/develop/main.gms#L327) the original [def 6.5$/kg = 0.2 $/Kwh](https://github.com/remindmodel/remind/blob/b9b8c3c90e378892e9b8f93c7a5f1876481a0f37/main.gms#L517) was strangely substituted.

You are right, the core reason is the code in the prepare-scripts. So maybe @LaviniaBaumstark can tell us if this can be easily adapted to avoid such mess in the future, or whether we should maybe agree on at least temporarily dispense with `/` in parameter descriptions.

Reply:
> I passed hours last year until I found out that a careless `/` in the description of a new switch [here](https://github.com/remindmodel/remind/commit/e4b5beb2860b020ac5e53939ccbc8b9570dcd2ba#diff-475ac83ca355b537a8b9c25d17290cea6bb65d000e13fdf7df4632520155f4e2R710) caused several lines of code to vanish. 

Should have written a bug report then.  pik-piam/lucode2#88

> Michaja, I really don't understand your reaction. 

Threat the disease, not the symptoms.  Simple as that.